config/webpack.dev.config.js

```diff
@@ -45,6 +45,8 @@ function generateWebpackDevConfig(buildOptions) {
       modules: false,
       warnings: true,
     },
+    // Needed to write the landing pages to disk so webpack-dev-server will actually serve them
+    writeToDisk: true,
```

Comment:
I'm actually not sure why it was working before, if I'm honest. Something something Metalsmith... :man_shrugging: 

If anybody knows how to _not_ write everything to disk during a `watch` task but still get `webpack-dev-server` to actually serve the pages created by `html-webpack-plugin`, I'm all ears! I hope this doesn't affect hot module reloading or any other fancy Webpack whatnots we may want to try out down the road.